charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -240,7 +240,7 @@ spec:
         - mountPath: /consul/login
           name: consul-data
           readOnly: false
-        {{- if not (and .Values.externalServers.enabled .Values.externalServers.useSystemRoots) }}
+        {{- if or (not (and .Values.externalServers.enabled .Values.externalServers.useSystemRoots)) .Values.client.enabled }}
```

Comment:
This init container should only talk to servers now, so we probably don't need this additional `or` here.